sktime/transformations/series/theta.py

```diff
@@ -66,7 +67,10 @@ def transform(self, Z, X=None):
         theta_lines = np.zeros((z.shape[0], len(theta)))
         for i, theta in enumerate(theta):
             theta_lines[:, i] = _theta_transform(z, trend, theta)
-        return pd.DataFrame(theta_lines, columns=theta)
+        if isinstance(self.theta, (float, int)):
+            return theta_lines
+        else:
+            return pd.DataFrame(theta_lines, columns=[self.theta])
```

Comment:
```suggestion
            return pd.DataFrame(theta_lines, columns=theta, index=y.index)
```
We can use the validated `theta` here.